test/extended/builds/gitauth.go

```diff
@@ -1,6 +1,6 @@
 package builds
 
-// these tests are diabled because the xip.io dns hook was proving way too unreliable;
+// these tests are diabled because the nip.io dns hook was proving way too unreliable;
```

Comment:
We'd need to enable it and see. But I would create separate issue to look into that.